.template/addons/openapi/package.json.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Create package.json file
+unless File.exist?('package.json')
+  create_file 'package.json',
+              <<~JSON
+                {
+                  "name": "#{APP_NAME}",
+                  "private": "true"
+                }
+              JSON
+end
+
+run 'yarn add --dev eslint'
+run 'yarn add --dev @nimblehq/eslint-config-nimble@2.2.1'
+run 'yarn add --dev @apidevtools/swagger-cli@4.0.4'
+run 'yarn add --dev @stoplight/spectral-cli@6.8.0'
+run 'yarn add --dev eslint-plugin-yml@1.8.0'
```

Comment:
```suggestion
run 'yarn add --dev @nimblehq/eslint-config-nimble@latest'
run 'yarn add --dev @apidevtools/swagger-cli@4.0'
run 'yarn add --dev @stoplight/spectral-cli@6.8'
run 'yarn add --dev eslint-plugin-yml@1.8'
```

Should we:
- use `latest` for our ESLint config? (if yes, please let's also update the Web variant 🙇 )
- use the latest patch version for the external packages? 🤔

Reply:
Thanks for pointing this out, I updated it in a943e33 ✅

Reply:
Okay, let's keep 2.2.1. I will create an issue to have a dedicated PR: the current version is 2.6, there are probably many lint to be fixed manually 🙈

Reply:
Actually there are already stories related with that: https://github.com/nimblehq/rails-templates/issues/422